src/os/shared/osapi-common.c

```diff
@@ -161,6 +162,13 @@ int32 OS_API_Init(void)
       return_code = OS_ERROR;
    }
 
+   microSecPerSec = OS_SharedGlobalVars.MicroSecPerTick * OS_SharedGlobalVars.TicksPerSecond;
+	
+   if ( microSecPerSec != 1000000 )
+   {   
+      OS_DEBUG("Error: Microsecs per sec value of %u does not equal 1000000 \n", microSecPerSec);
```

Comment:
In this debug statement I'd rather want to know the raw values (MicroSecPerTick, TicksPerSecond) as opposed to the multiplication product.

Also, these are `uint32`, so for printing should use the `%lu` format code and cast it to `(unsigned long)`.